content/blog/new-post-mutable-vs-immutable/index.md

```diff
@@ -0,0 +1,103 @@
+---
+{
+	title: "Mutable vs Immutable Data Types",
+	description: "Using mutable data types can be dangerous in multi-threaded applications. To help that we can make sure of thread safer immutable data types",
+	published: '2022-07-20T16:56:03.000Z',
+	authors: ['alexchadwick'],
+	tags: ['typescript', 'data structures', 'computer science'],
+	attached: [],
+	license: 'cc-by-nc-sa-4'
+}
+---
+
+# Mutable vs Immutable Data Types
```

Comment:
![image](https://user-images.githubusercontent.com/9100169/181438235-e5239ebc-7f87-4bff-859c-ef4c94e4c0cb.png)

Since the `title` field is already an `h1`, it's confusing to have it duplicated. Let's remove this H1 and make the lower elements `h1`s instead

Reply:
```suggestion
```